addons/website_sale_loyalty/tests/test_shop_sale_coupon.py

```diff
@@ -91,6 +91,7 @@ def test_01_admin_shop_sale_loyalty_tour(self):
         })
 
         self.env.ref("website_sale.search_count_box").write({"active": True})
+        self.env.ref("website_sale.reduction_code").write({"active": True})
```

Comment:
@rdeodoo Yes we have several others tours that test the customize options by clicking on the "customize" tab in edit mode...
If the option is now enabled first in Python, it is so that this tour can work in frontend without having to enable the option in edit mode. Do you want us to change something here in the end?

Reply:
Indeed, I had a look and `shop_customize` is doing that now :+1: 
Nothing to do then